ToDoApp/ToDoApp/ToDoListViewModel.swift

```diff
@@ -85,17 +87,19 @@ final class ToDoListViewModel: ObservableObject {
                                     toDoItems: todos.other))
                 }
                 state = .loaded(todoSections)
+                loading = false
```

Comment:
shouldn't false be assigned regardless of the result?

Reply:
I thought after the list of tasks are successfully loaded, then it'll not be loading anymore, so it will be false.

Reply:
I mean that loading is completed even when task service returns an error. Isn't it?

Reply:
Yes, I have fixed this